### ISSUE TITLE
aarch64 docker build

### ISSUE THREAD

**Issue by net0-dev** [euzu/m3u-filter#138]
Could you please add an automated aarch64 docker build for those of us using the oracle free tier?

**Comment by euzu** [comment]
This is comming with the next release.

**Comment by 36-chambers** [comment]
In the meantime I got it working with this in the Dockerfile

```
RUN cargo build --release --target $(uname -m)-unknown-linux-gnu
RUN mv "/src/target/$(uname -m)-unknown-linux-gnu/release/m3u-filter" /m3u-filter
```

**Comment by euzu** [comment]
@36-chambers yes this is possible because you are compiling on the target plattform. And i think the output of your `uname -m` is `aarch64`.

**Comment by 36-chambers** [comment]
I'm using it with my docker compose I have deployed on the arm box so that the docker compose will build whatever it needs. On my laptop it's x86. On my arm server it's aarch64.
